processor/otel/metrics.go

```diff
@@ -371,6 +376,16 @@ func numberSample(dp pdata.NumberDataPoint, metricType model.MetricType) (model.
 	}, true
 }
 
+func summarySample(dp pdata.SummaryDataPoint) (model.MetricsetSample, bool) {
```

Comment:
is there a reason for the `bool`? seems like that could be dropped, or the struct declaration even inlined.

Reply:
Just being a bit lazy and copy-pasting the code patterns for other metric types. I'll drop the bool.